carma_record/launch/carma_record.launch

```diff
@@ -0,0 +1,19 @@
+<!--
+  Copyright (C) 2021 LEIDOS.
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<launch>
+  <!-- The carma_record_node will load in the parameters from the rosparam server, and export a exclude_regex param -->
+  <node name="carma_record_load_regex" pkg="carma_record" type="carma_record_node" output="screen"/>
```

Comment:
don't set output="screen" unless using it for testing. This will prevent logging to a file which is preferable when trying to debug in the future.